Makefile

```
@@ -1,256 +1,188 @@
-		--env-file envvars \
-		$(IMAGE_NAMESPACE)/$(IMAGE_NAME):$(IMAGE_TAG) python main
-
-.PHONY: run
-run: setup re-run
-
-.PHONY: shell-run
-shell-run: set-run-variable-values
-	podman run --rm -ti --volume $(PWD):/mnt/code:rw \
-		--pod $(POD_NAME) \
-		--env PYTHONPATH=/mnt/code \
-		--env-file envvars \
-		$(IMAGE_NAMESPACE)/$(IMAGE_NAME):$(IMAGE_TAG) bash
+setup: cluster reinstall-stack 
+
+.PHONY: reinstall-stack
+reinstall-stack: helm-repo tika minio postgresql elasticsearch #opensearch
+	$(k8s) wait --for=condition="Ready" pods -A --all -l job-name!=minio-provisioning
+
+.PHONY: install-pipeline
+install-pipeline: 
+	$(helm_install) --namespace querido-diario --create-namespace \
+		--set elasticsearch.password=$(shell $(k8s) get secret querido-diario-elasticsearch-es-elastic-user -n default -o jsonpath="{.data.elastic}" | base64 -d) \
+		--set textExtractionJob.image="$(IMAGE_NAMESPACE)/$(IMAGE_NAME):$(IMAGE_TAG)" \
+		querido-diario-pipeline charts/querido-diario-pipeline/
+
+.PHONY: uninstall-pipeline
+uninstall-pipeline: 
+	- $(helm) uninstall --namespace querido-diario querido-diario-pipeline
+
+.PHONY: credenciais
+credenciais: 
+	@echo "Essas são as credenciais para você acessar os serviços rodando no cluster local:"
+	@echo POSTGRES_PASSWORD = $(shell kubectl get secret --namespace postgresql postgresql -o jsonpath="{.data.password}" | base64 -d)
+	@echo POSTGRES_ADMIN_PASSWORD=$(shell kubectl get secret --namespace postgresql postgresql -o jsonpath="{.data.postgres-password}" | base64 -d)
+	@echo MINIO_ROOT_USER=$(shell $(k8s) get secret --namespace minio minio -o jsonpath="{.data.root-user}" | base64 -d)
+	@echo MINIO_ROOT_PASSWORD=$(shell $(k8s) get secret --namespace minio minio -o jsonpath="{.data.root-password}" | base64 -d)
+	@echo ELASTICSEARCH_PASSWORD=$(shell $(k8s) get secret querido-diario-elasticsearch-es-elastic-user -n default -o jsonpath="{.data.elastic}" | base64 -d)
+
+DIARIOS := 1302603/2023-08-16/eb5522a3e160ba9129bd05617a68badd4e8ee381.pdf 3304557/2023-08-17/00e276910596fa4b4b7eb9cbec8a221e79ebbe0e 4205407/2023-08-10/c6eb1ce23b9bea9c3a72aece0e762eb883a8a00a.pdf 4106902/2023-08-14/b416ef3008654f84e2bee57f89cfd0513f8ec800 2611606/2023-08-12/7b010f0485bbb3bf18500a6ce90346916e776d62.pdf
+$(DIARIOS):
+	@if [ ! -f $(join $(DIARIOS_DIR),/$@) ]; then \
+		echo "Baixando $@"; \
+		curl -XGET --output-dir $(DIARIOS_DIR) --create-dirs --output $@ $(QUERIDO_DIARIO_CDN)$@; \
+	fi 
+	
+.PHONY: prepara-ambiente
+prepara-ambiente: expoe-servicos diarios base-de-dados derruba-servicos
+
+.PHONY: expoe-servicos
+expoe-servicos: derruba-servicos
+	@nohup $(k8s) port-forward --namespace minio svc/minio 9000:9000 > /dev/null 2>&1 &
+	@nohup $(k8s) port-forward --namespace minio svc/minio 9001:9001 > /dev/null 2>&1 &
+	@nohup $(k8s) port-forward --namespace postgresql svc/postgresql 5432:5432 > /dev/null 2>&1 &
+	@echo "Minio esta disponivel na porta 9000 e 9001"
+	@echo "Postgresql está disponivel na porta 5432"
```

Como por enquanto exigimos que se tenha postgres na máquina, é melhor já deixar o mapeamento pra rede local numa porta diferente da 5432 pra não dar conflito.
```suggestion
	@nohup $(k8s) port-forward --namespace postgresql svc/postgresql 5433:5432 > /dev/null 2>&1 &
	@echo "Minio esta disponivel na porta 9000 e 9001"
	@echo "Postgresql está disponivel na porta 5433"
```